### DISCUSSION TITLE
New component - "ColorPicker"

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2707] [Ideas]
I would like this library to have one more data-entry component - ColorPicker.
input type="color" does not support transparent or opacity, but i want to specify that also.

So now i have my custom component:
```
function ColorPicker() {
    const [value, setValue] = React.useState('');

    return (
        <InputGroup style={{width: 180, height: 40}}>
            <Input value={value} onChange={setValue} type="color" style={{height: "100%"}}/>
            <Input value={value} onChange={setValue}/>
        </InputGroup>
    )
}
```
This is how it looks
![Снимок экрана (61)](https://user-images.githubusercontent.com/78722918/189346669-9de3bc01-4e20-4fe7-9a6f-90c98e39cf6b.png)

But as i already told you, it does not support alpha chanel.

**Comment by simonguo** [comment]
https://rsuitejs.com/resources/extensions/
There are some extended components, ColorPicker is one of them. 

<img width="364" alt="image" src="https://user-images.githubusercontent.com/1203827/189379342-05295a86-f533-46b7-ae53-e5985bb7f943.png">

https://cxiaof.github.io/rsuite-color-picker/assets/
